frontend/src/reducers/content.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from "@reduxjs/toolkit"
+
+const content = createSlice({
+    name: "content",
+    initialState: {
+        items: [],
+        error: null
+    },
+    reducers: {
+        setItems: (store, action) => {
+            store.items = action.payload
+        },
+        setError: (store, action) => {
+            store.error = action.payload
+        }
```

Comment:
Not quite sure what this does but will have to do some research and look this up. This was new to me.